src/routes.ts

```diff
@@ -29,7 +29,10 @@ routes.post("/", async (req, res) => {
     log.debug(`Received Bundle: ${JSON.stringify(bundle)}`);
 
     const bundleId = `${Number(bundle.blockNumber)}_${request.id}`;
-    await aws.upload(bundle, bundleId);
+    // Context on spelling https://www.sistrix.com/ask-sistrix/technical-seo/http/http-referrer/
+    const referrer: string =
+      req.headers.referer || (req.headers.referrer as string);
```

Comment:
so we allow both spellings? 

ubernit - I would think the left side should be the correct spelling and the right one be the accepted alternative. Like this...

```suggestion
    const referrer: string =
      req.headers.referrer || (req.headers.referer as string);
```